.travis.yml

```diff
@@ -9,6 +9,14 @@ rvm:
   - 2.3.0
   - ruby-head
   - rbx-2
+  - jruby-9.0.5.0
+  - jruby-head
+
+jdk:
+  - oraclejdk8
+
+before_install:
+  - '[ "$JRUBY_OPTS" != "" ] && export JRUBY_OPTS="-Xcompat.version=2.0 --server -Xcompile.invokedynamic=false -Xcli.debug=true --debug"'
```

Comment:
plz change to `9.0.4.0` since travis caches it, faster, and JRUBY_OPTS to `'--dev --debug --Xcli.debug=true'`. see https://github.com/rails/rails/pull/23499#discussion_r55161731